rednotebook/external/text2rednotebook.py

```diff
@@ -0,0 +1,145 @@
+"""
+Mass import diary/journal entries from markdown/plain text/odt into rednotebook
```

Comment:
Can you add some more details about the expected form of the input and output?

Reply:
Done